Sources/TuistGenerator/Generator/DescriptorGenerator.swift

```diff
@@ -0,0 +1,85 @@
+import Basic
+import Foundation
+import TuistCore
+import TuistSupport
+import XcodeProj
+
+public struct GeneratedSchemeDescriptor {
```

Comment:
What about renaming all the entities defined here to be `Generate...` instead of `Generated...`. I think using the participle is a bit misleading because it represents something that we are passing around until it reaches the component that materialises it.

Reply:
I was thinking perhaps even dropping the prefix altogether as they all have a `Descriptor` suffix?